packages/@react-aria/focus/src/FocusScope.tsx

```diff
@@ -45,7 +45,8 @@ export interface FocusManagerOptions {
   /** Whether focus should wrap around when it reaches the end of the scope. */
   wrap?: boolean,
   /** A callback that determines whether the given element is focused. */
-  accept?: (node: Element) => boolean
+  accept?: (node: Element) => boolean,
+  preview?: boolean
```

Comment:
needed so i could use the same setup and determine where focus would move in both the tabbable and focusable case next before actually committing to it